### ISSUE TITLE
Create About Us page

### ISSUE THREAD

**Issue by deweydell** [WriteSpeakCode/writespeakcode.com#17]
### Description

We want to show who the organizers are and what their role is.

**Why this is valuable:**

This will help people get a sense of who runs the community and give organizers an ability to feel seen as a thank you for all the hard work. 

### Acceptance Criteria

- [ ] There is a page at writespeakcode.com/about-us
- [ ] The Page has a separate section for photos/bios in this order: board members,  chapter leaders, organizers, and website contributors.
- [ ] The data for the page is managed in a data file and pulled in via a page query.
- [ ] Each person has a headshot, name, role, and social links.
- [ ] Only name and role are required.

### Development Plan
- Design and data needed.

**Comment by deweydell** [comment]
@nayonnapurnell I'm happy to take the lead on the design and the gathering of organizer data on this one. The other next step in this one is to add the organizer content type to the CMS config file - do you want to take that on?

**Comment by nayonnapurnell** [comment]
Yes.  I am okay with that.  So, I don’t need to create the page?  I just need to add content right?

On May 7, 2019, at 2:18 PM, Nicola B. <notifications@github.com<mailto:notifications@github.com>> wrote:


@nayonnapurnell<https://github.com/nayonnapurnell> I'm happy to take the lead on the design and the gathering of organizer information on this one. The other next step in this one is to add the organizer content type to the CMS config file - do you want to take that on?

**Comment by deweydell** [comment]
I turned the About page issue into an epic and am going to break it up into smaller tickets. I assigned. you to the issue above ^ which is the smaller task you can get started on while I gather information and draw up the design.

**Comment by deweydell** [comment]
For designs I'm thinking something like this similar to the sponsors section on the landing page:
![Screen Shot 2019-05-15 at 10 48 10 PM](https://user-images.githubusercontent.com/11186572/57823872-163dda80-7767-11e9-9f4f-9450e68fdc6a.png)

For each person something like this. This would require adding an optional city to the organizer data.
![Screen Shot 2019-05-15 at 11 08 35 PM](https://user-images.githubusercontent.com/11186572/57823873-163dda80-7767-11e9-8a07-5951b8a73a24.png)

I also think it would be nice to put the conference organizers for 2019 on the conference page too. This would require adding a list of conference years to the organizer data.

**Comment by halfghaninne** [comment]
@deweydell Do you envision this being a distinct page or a section of the landing page? Right now, I've set it up as the latter.

Also, any update to designs? I think I remember the sponsorship design changing a bit between May and the conference. If not, no worries. I can give it a go and we can revise in review.

**Comment by deweydell** [comment]
@halfghaninne I would like it to be a separate page! It will eventually contain everyone involved in the org, so it will be pretty big.

Re: design. Can you start working on it just getting the data up and rendering the people in each section? I can draw up some screenshots of what I think it should look like, because yeah I have some new thoughts now. Part of the dev approach also can be gathering some inspiration from other about pages/team pages that we like.

**Comment by deweydell** [comment]
@halfghaninne some sketches for the design implementation! Though for the image, I'd like to use the photo of all of us on stage at the 2019 conference. For the first iteration, let's implement something like what we did for 2017/2018 organizers (but with navy text on white background and link styles that match our current design).

I like the idea of "cards" or something that can show more info for board members, chapter leaders and main organizers, but that can be 2nd iteration.

![Screen Shot 2019-05-15 at 11 12 53 PM](https://user-images.githubusercontent.com/11186572/67149281-7e916a80-f277-11e9-97a5-145b4bc57504.png)

 

![Screen Shot 2019-10-12 at 2 34 46 PM](https://user-images.githubusercontent.com/11186572/67149252-23f80e80-f277-11e9-843e-5024dd5aab7b.png)
![Screen Shot 2019-10-12 at 2 35 55 PM](https://user-images.githubusercontent.com/11186572/67149253-23f80e80-f277-11e9-8e36-399970fce4fe.png)

**Comment by deweydell** [comment]
Here's the photo I like for the main page

![WSC2019-493](https://user-images.githubusercontent.com/11186572/67149359-5b1aef80-f278-11e9-9900-a20cd3740ec1.jpg)

**Comment by deweydell** [comment]
@halfghaninne do you need any help with this page?

**Comment by halfghaninne** [comment]
@deweydell Sorry I didn't see your message from a couple weeks ago! No help per se, though it would be helpful to have access to netlify logs to see why my PR builds are failing! I'd also love some feedback on the look of the page so far.

Oh! I also have a question on page interactivity. I've left out bios on the first pass at the organizer components, but we've collected the information. Do we want to include an extended view of each organizer that includes their bio (similar to the speaker dialogue part of the 2019 conference page)? Or do you want to leave it out for this iteration?

**Comment by deweydell** [comment]
Ok nw, I look into adding you to netlify and check out your page locally and leave some feedback.
Let's leave bios out for now. In the meantime DMing you the netlify log failures on slack.
